CHANGELOG.md

```diff
@@ -16,6 +16,40 @@ All notable changes to this project will be documented in this file.
 > ### Bugfixes
 > N/A
 
+# v3.2.0
+
+- **SwifterSwift**
```

Comment:
Can we label this section **API Breaking**?

Reply:
Sure!